tests/kernel/wave/wave_attention_test.py

```diff
@@ -193,6 +193,8 @@ def repeat(
         schedule=enable_scheduling,
         use_scheduling_barriers=enable_scheduling_barriers,
     ):
+        torch.cuda.empty_cache()
```

Comment:
what do you think of moving this to a 
```
initialize_test(seed=0)
```
function in utils.py?

Reply:
as in a function that has both torch.cuda.empty_cache() + torch.manual_seed(seed)?

Reply:
done, but renamed to `initialize_seed_and_cache` for better explanation to what it does. :)